src/Microsoft.AspNetCore.Mvc.Core/Formatters/TextInputFormatter.cs

```diff
@@ -15,6 +16,53 @@ namespace Microsoft.AspNetCore.Mvc.Formatters
     /// </summary>
     public abstract class TextInputFormatter : InputFormatter
     {
+        // ASP.NET Core MVC 1.0 used Encoding.GetEncoding() when comparing the request charset with SupportedEncodings.
+        // That method supports the following alternate names for system-supported encodings. This table maps from
+        // .NET Core's keys in s_encodingDataTable to values in s_encodingDataTableItems, less identity mappings. It
+        // should be kept in sync with those Unix-specific tables in
+        // https://github.com/dotnet/coreclr/blob/master/src/mscorlib/src/System/Globalization/EncodingTable.Unix.cs
+        // or their Windows equivalents (also for .NET Core not desktop .NET): COMNlsInfo::EncodingDataTable and
+        // COMNlsInfo::CodePageDataTable in EncodingData in
+        // https://github.com/dotnet/coreclr/blob/master/src/classlibnative/nls/encodingdata.cpp
+        private static readonly IReadOnlyDictionary<string, string> EncodingAliases = new Dictionary<string, string>
```

Comment:
I note your extreme disappointment that this table contains only `32` entries but have to move beyond it. The original `42` in `s_encodingDataTable` contained `10` entries that mapped to `Encoding`s with the given name (i.e. identity mappings).